features/product/viewing_products/viewing_product_from_specific_taxon_sorted_by_position.feature

```diff
@@ -18,14 +18,14 @@ Feature: Viewing products from specific taxon sorted by position
         And this product is in "Toys" taxon at 2nd position
         And this product is in "Soft Toys" taxon at 2nd position
 
-    @ui
+    @api @ui
     Scenario: Seeing sorted product on list from a specific taxon
-        When I browse products from taxon "Toys"
+        When I browse products from product taxon code "Toys"
```

Comment:
Why did you change the step definition?

Reply:
Because previous definition was from this method: `\Sylius\Behat\Context\Api\Shop\ProductContext::iBrowseProductsFromTaxon`
And I needed to change filter 'taxon' to 'productTaxons.taxon.code'. I've created new method because I didn't want to break other tests using iBrowseProductsFromTaxon method

Reply:
Now I wonder why we have two similar filters defined: `taxon` and `productTaxons.taxon.code`, probably we should have defined only one of them (`taxon` I guess). What is more, there is a filter definition https://github.com/Sylius/Sylius/blob/1.13/src/Sylius/Bundle/ApiBundle/Filter/Doctrine/TaxonFilter.php and you should apply the fix also there and of course test it properly.

Reply:
@lchrusciel do you have any opinion on these filters? 👀

Reply:
I've decided to use filter with `productTaxons.taxon.code` because I found it here: https://master.demo.sylius.com/api/v2/docs
In section 'product' there are two params 'taxon' or 'productTaxons.taxon.code' and when I was using 'taxon' it didn't return any product (It worked only with 'productTaxons.taxon.code') so that's why I used second param (filter)